docs/UserGuide.md

```diff
@@ -32,155 +32,33 @@ Reache is a desktop app that helps busy working professionals manage their large
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Features
+## 2. Features
 
-<div markdown="block" class="alert alert-info">
 
-**:information_source: Notes about the command format:**<br>
+## 3. Command Summary
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+| **Action** | **Format, Example** |
+| --- | --- |
+| **Add** | Format: `add <name>` <br> Example: `add Alex Dunphy` |
```

Comment:
I think to keep it consistent we should change name to NAME. Same applies for the Contact Field Summary section.